src/node/blockstorage.cpp

```diff
@@ -298,6 +298,7 @@ void BlockManager::FindFilesToPrune(
     // Distribute our -prune budget over all chainstates.
     const auto target = std::max(
         MIN_DISK_SPACE_FOR_BLOCK_FILES, GetPruneTarget() / chainman.GetAll().size());
+    const uint64_t target_sync_height = chainman.m_best_header->nHeight;
```

Comment:
nit: Any reason this declaration is up here and not directly above where it is used below?